common/Makefile.hal

```diff
@@ -33,6 +33,9 @@ or if you have a Proxmark 3 RDV4 with the BT add-on:
 PLATFORM=PM3RDV4
 PLATFORM_EXTRAS=BTADDON
 
+Option to choose standalone mode, e.g. HF_COLIN
+SANDALONE=HF_COLIN
+
```

Comment:
spelling?
and default standalone mode is sammy_run.

Reply:
Yes sorry.
I've changed to LF_SAMYRUN in examples.